scripts/dataflow_copy.py

```diff
@@ -36,15 +95,17 @@
         for session in sessions:
             # set source dir - NAS
             trg_dir = os.path.join(src_dir, session)
-            # set dest dir - neo if subject id odd, else drwho
-            subj_id = int(session.split('_')[0])
-            if subj_id % 2: #odd
-                dest_dir = os.path.join(dest_dir_1, session)
-            else: #even
-                dest_dir = os.path.join(dest_dir_2, session)
-
-            # Note: trg_dir and dest_dir do not have trailing slashes here!
-            copy_files(trg_dir, dest_dir, db_table, sensor_file_table)
+            # check if session is already copied
+            is_copied, dest_dir = check_if_copied(session, suitable_volumes)
+            if not is_copied:
+                dest_dir = os.path.join(get_volume_to_fill(suitable_volumes, reserve_threshold), session)
+                print(f'copying new session {session} to {dest_dir}')
```

Comment:
consider logging instead of prints

Reply:
This is on the cards and I will spend time on a full logging overhaul of the the dataflow write/copy/delete scripts next.